metagraph/src/annotation/representation/column_compressed/annotate_column_compressed.cpp

```diff
@@ -234,27 +234,39 @@ void ColumnCompressed<Label>::serialize_counts(const std::string &filename) cons
 
     for (size_t j = 0; j < relation_counts_.size(); ++j) {
         if (!relation_counts_[j].size()) {
-            sdsl::int_vector<>(bitmatrix_[j]->num_set_bits(), 0, count_width_).serialize(out);
+            sdsl::int_vector<>(bitmatrix_[j]->num_set_bits(), 0, 1).serialize(out);
```

Comment:
Explain with a comment that the serialization format is: bit vector, counts. It'll be easier to understand what's happening here

Reply:
done